backend/models.py

```diff
@@ -42,7 +64,7 @@ class Participant(db.Model):
     first_name = Column(String(50), nullable=False)
     last_name = Column(String(50), nullable=False)
     email = Column(String(200), unique=True, nullable=False)
-    github = Column(String(200), unique=True, nullable=False)
+    github_username = Column(String(200), unique=True, nullable=False)
```

Comment:
That change had not been included in alembic migration file, and in db schema github remains, which brakes most of a changes from that PR.